### DISCUSSION TITLE
Minecraft World File Import to Server

### DISCUSSION THREAD

**Discussion by Stewart-Michael-116** [AlexProgrammerDE/balena-minecraft-server#155] [Q&A]
Hello! How can I put a local world file I have onto the server. This would let me be able to import worlds into the server. I have looked at the documentation and tried replacing the world folder data with my world's data, but it just created a new world. Any thoughts?

**Comment by viedok** [comment]
After a bit of searching, I found that you should import your singleplayer world in the server files, delete the `world` folder and rename your singleplayer world folder `world`.
Let me know if this doesn't work! :)
